packages/cardano-services/src/ChainHistory/DbSyncChainHistory/types.ts

```diff
@@ -105,7 +107,7 @@ export interface WithdrawalModel {
 
 export interface RedeemerModel {
   index: number;
-  purpose: 'cert' | 'mint' | 'spend' | 'reward' | 'voting' | 'proposing';
+  purpose: 'cert' | 'mint' | 'spend' | 'reward' | 'vote' | 'propose';
```

Comment:
Are db-sync `master` branch schema docs incorrect?
![image](https://github.com/user-attachments/assets/e99649d0-66e7-4628-be94-32bc5c4285d9)

Reply:
IDK, I didn't checked...

Reply:
What's the reason for https://github.com/input-output-hk/cardano-js-sdk/pull/1480/commits/71f6de7c28a568c66544515316e82384bcf5e442 commit then? Which db-sync version it is?

Reply:
db-sync version is 13.5.0.1
I taken the values directly from the DB

Reply:
13.5.0.1 tagged commit also has `voting` and `proposing` in docs. Let's report to db-sync devs